packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -828,6 +1017,20 @@ String? _checkAndEncode(dynamic script) {
   return script.substring(_kWebScriptPrefix.length, script.length - 2) as String?;
 }
 
+// This function remove all file in test output directory
+Future<void> _cleanTestOutputFiles() async {
+  final Directory fileTestOutputsDirectory = Directory(testOutputsDirectory);
+  // Sometime file is not exists
+  try {
+    final List<FileSystemEntity> files = fileTestOutputsDirectory.listSync();
+    for (final FileSystemEntity file in files) {
+      if (FileSystemEntity.isFileSync(file.path)) {
+        await File(file.path).delete();
+      }
+    }
+  } catch(_) {}
```

Comment:
what errors is this attempting to catch?

Reply:
this code is redundant, i removed.